standalone-metastore/metastore-server/pom.xml

```diff
@@ -311,6 +311,22 @@
       <artifactId>curator-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.nimbusds</groupId>
+      <artifactId>nimbus-jose-jwt</artifactId>
+      <version>9.20</version>
+    </dependency>
+    <dependency>
+      <groupId>org.pac4j</groupId>
+      <artifactId>pac4j-core</artifactId>
+      <version>4.5.5</version>
```

Comment:
Can you define the versions at the top instead of hard-coding them here?

Reply:
Sure. Thanks for the suggestion.